common/src/main/resources/tachyon-default.properties

```diff
@@ -40,6 +40,9 @@ tachyon.test.mode=false
 tachyon.max.table.metadata.byte=5MB
 tachyon.host.resolution.timeout.ms=5000
 
+# Security properties
+tachyon.security.authentication=NOSASL
```

Comment:
Yes, `tachyon.security.username` is specified only when you need a username and `tachyon.authentication.provider.custom.class` is specified only if `tachyon.security.authentication=CUSTOM`